packages/components/pie-checkbox/src/checkbox.scss

```
@@ -297,6 +315,11 @@
                     var(--dt-color-interactive-brand-s),
                     calc(var(--dt-color-interactive-brand-l) - var(--dt-color-hover-01))
                 );
+
+                @supports (background-color: color-mix(in srgb, black, white)) {
+                    --checkbox-bg-color: color-mix(in srgb, var(--dt-color-hover-01-bg) var(--dt-color-hover-01),  var(--dt-color-interactive-brand));
+                    --checkbox-border-color: color-mix(in srgb, var(--dt-color-hover-01-bg) var(--dt-color-hover-01),  var(--dt-color-interactive-brand));
```

It looks like we set the border colour the same as the background colour in quite a few places, could we do something like this?
```suggestion
                    --checkbox-border-color: var(--checkbox-bg-color);
```